charmcraft.yaml

```diff
@@ -6,24 +6,38 @@
 type: "charm"
 bases:
   - build-on:
-    - name: "ubuntu"
-      channel: "22.04"
+      - name: "ubuntu"
+        channel: "22.04"
     run-on:
-    - name: "ubuntu"
-      channel: "22.04"
+      - name: "ubuntu"
+        channel: "22.04"
 parts:
   charm:
     build-packages:
-    - build-essential
-    - cargo
-    - libffi-dev
-    - libssl-dev
-    - libxml2
-    - libxslt1-dev
-    - pkg-config
-    - python3-lxml
-    - python3-openssl
-    - python3-pip
-    - python3-setuptools
-    - python3-wheel
-    - rustc
+      - libxml2-dev
```

Comment:
Would you mind ordering these alphabetically?

Reply:
Updated, thanks!